db/dump.py

```diff
@@ -238,6 +241,56 @@ def generate_where(row_name, start_t=start_time, end_t=end_time):
         connection.close()
 
 
+def update_sequence(seq_name, table_name):
+    """ Update the specified sequence's value to the maximum value of ID in the table.
+
+    Args:
+        seq_name (str): the name of the sequence to be updated.
+        table_name (str): the name of the table from which the maximum value is to be retrieved
+    """
+    with db.engine.connect() as connection:
+        connection.execute(text("""
+            SELECT setval('{seq_name}', max(id))
+              FROM {table_name}
+        """.format(seq_name=seq_name, table_name=table_name)))
+
+
+def update_sequences():
+    """ Update all sequences to the maximum value of id in the table.
+    """
+    # lowlevel_id_seq
+    logging.info('Updating lowlevel_id_seq...')
```

Comment:
I've created https://tickets.metabrainz.org/projects/AB/issues/AB-364 to add some docs to remind us that we have to update this every time we make a new table.